plugins/modules/netcup_dns.py

```diff
@@ -53,7 +53,7 @@
   type:
     description:
       - Record type.
```

Comment:
```suggestion
      - Record type.
      - Support for V(OPENPGPKEY), V(SMIMEA), and V(SSHFP) was added in community.general 8.1.0.
```

Reply:
Maybe also mention which versions of nc_dnsapi are needed for these options. While `OPENPGPKEY` and `SMIMEA` have been supported for quite some time now (according to the PR you linked to), `SSHFP` only has been added earlier this year.